include/avif/avif.h

```diff
@@ -526,7 +526,10 @@ typedef struct avifReformatState
     int rgbMaxChannel;
     float yuvMaxChannelF;
     float rgbMaxChannelF;
-    int uvBias; // the integer value of 0.5 for the appropriate bit depth [128, 512, 2048]
+    float yBias; // minimum Y value
```

Comment:
I suggest we remove the unused member rgbDepth and change yuvMaxChannelF to a local variable.